modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/asset/model/ObjectEntryFolderAssetRendererFactory.java

```diff
@@ -40,8 +41,16 @@ public AssetRenderer<ObjectEntryFolder> getAssetRenderer(
 			long classPK, int type)
 		throws PortalException {
 
-		return new ObjectEntryFolderAssetRenderer(
-			_objectEntryFolderService.getObjectEntryFolder(classPK));
+		ObjectEntryFolder objectEntryFolder =
+			_objectEntryFolderService.getObjectEntryFolder(classPK);
+
+		if (!FeatureFlagManagerUtil.isEnabled(
```

Comment:
if you put this first , you are saving the get and you can omit the variable.

Reply:
I want to get it first in order to see if exists and then I use it to get the companyId

Reply:
where?

Reply:
ok, forget my comment, I did not saw, it now.

<img src="https://media4.giphy.com/media/13d2jHlSlxklVe/giphy.gif?cid=bd3ea57e1srp2c5y3n6sw5ctm4sqekjtywsz917u0f1a9st5&ep=v1_gifs_search&rid=giphy.gif&ct=g"/>

Reply:
post lunch brain...